.env

```diff
@@ -11,3 +11,8 @@ MYSQL_PORT="3306"
 MYSQL_USER="root"
 MYSQL_PASSWORD="secret"
 MYSQLDUMP_DATABASE="my_wiki"
+# specify the frequencies needed
```

Comment:
@vijayabharathib Here this has to be like `key=value` format. 

```
SCHEDULE_BACKUP_HOURLY="true"
```
like the above

Reply:
done.